mkdocs/topics/testing.md

```diff
@@ -0,0 +1,56 @@
+# Testing
+
+Tests enable you to **quickly** validate data across your project.  
+
+!!! note
+    Tests are run with the `visivo test` command (_[docs](../../reference/cli/#test)_).
+
+Tests can run any arbitrary python statement so long as it evaluates to `True` or `False`. Additionally tests can access trace data from across the project by using the `$(project)` context variable. By default trace data arrays will be represented as numpy arrays. 
+
+The combination of python based boolean statements and access to all trace data makes it _super fast_ all while enabling complex assertions about your data. 
+
+!!! example
+
+    === "Revenue Sums Match"
+   
+        ``` yaml
+        tests:
+          - name: revenue-date-grains-match
+            logic: "numpy.sum( $(project.traces['revenue-per-week'].props.y) ) = numpy.sum( $(project.traces['revenue-per-month'].props.y) )"
```

Comment:
Does this work with cohorts?  Since there are multiple "props.y"

Reply:
We'll probably need to re-write this testing docs page once we finalize our syntax for context. I don't think people would need to specify a cohort though, we can flatten out the trace data and make the cohort a column again.